snappi_ixnetwork/ixnetworkconfig.py

```diff
@@ -0,0 +1,190 @@
+import json
+import time
+import re
+from snappi_ixnetwork.timer import Timer
+from snappi_ixnetwork.logger import get_ixnet_logger
+import snappi
+
+class IxNetworkConfig(object):
+    _TYPE = {
+        "chassis_chains": {"type": "ChassisChainIter"}
+    }
+
+    def __init__(self, ixnetworkapi):
+        print(f"End {object.__class__}")
+        self._api = ixnetworkapi
+        self._chassis_chains = None
+    
+    @property
+    def chassis_chains(self):
+        if self._chassis_chains is None:
+            self._chassis_chains = ChassisChainIter()
+    
+        return self._chassis_chains
+
+    def config(self):
+        self.logger.info("Configuring IxNConfig")
+
+class ChassisChain(object):
+    _TYPES = {
+        "primary": {"type": str},
```

Comment:
It will good if we add docstring for all user configurable property similar as `snappi`. It will maintain same Specification as `sanppi` also help user to write script.